README.md

```diff
@@ -396,8 +396,8 @@ Some standouts:
 
 | Hardware   | Instructions                                                                                                    |
 |------------|-----------------------------------------------------------------------------------------------------------------|
-| CPU        | `pip install -U "jax[cpu]"`                                                                                       |
-| NVIDIA GPU on x86_64 | `pip install -U "jax[cuda12]"`        |
+| CPU        | `pip install -U jax`                                                                                            |
+| NVIDIA GPU | `pip install -U "jax[cuda12]"`                                                                                  |
```

Comment:
Should this just be `jax[cuda]`?

So that in the future when we have cuda 13.. jax[cuda] will install 13

and users who want to stay on 12 can do cuda[12] mentioned in the installation doc. 

I guess my question is: Where do we mention `jax[cuda]`?

Reply:
I don't really think `cuda` adds much benefit, but I didn't want to delete it. So I'm happy with leaving it undocumented.